tests/test_port_dpb_system.py

```diff
@@ -248,25 +242,380 @@ def test_port_breakout_with_acl(self, dvs):
 
         # Verify port is removed from ACL table
         self.dvs_acl.verify_acl_table_count(1)
-
-        #TBD: Uncomment this, or explain why Ethernet0 is being added back to ACL table
-        # Also, string "None" is being added as port to ACL port list after the breakout
-        #self.dvs_acl.verify_acl_group_num(0)
+        self.dvs_acl.verify_acl_group_num(0)
 
         # Verify child ports are created.
         self.verify_only_ports_exist(dvs, ["Ethernet0", "Ethernet1", "Ethernet2", "Ethernet3"])
 
-        # Enable below snippet after fixing the above issues
-        '''
         # Move back to 1x100G[40G] mode and verify current mode
         dvs.change_port_breakout_mode("Ethernet0", "1x100G[40G]", "-f")
         dpb.verify_port_breakout_mode(dvs, "Ethernet0", "1x100G[40G]")
-        '''
 
         # Remove ACL table and verify the same
         self.dvs_acl.remove_acl_table("test")
         self.dvs_acl.verify_acl_table_count(0)
 
+    def test_cli_command_with_hyphen_f_option(self, dvs):
```

Comment:
what abt test_cli_command_with_force_option ?

Reply:
Sorry, I dint get what test case you are talking about. Because I have covered both with and without "-f" option

Reply:
I was suggesting name for this function,  because single letter options are just for easy to use. Actual option is usually bigger and more readable. For example:
-f --> --force-remove-dependencies. 
So what about test_cli_command_with_force_option or test_cli_with_force-remove-dependencies()

Reply:
Makes sense. Let me update it.